File: traslate_py_C/example2.c
#include <stdio.h>

int main(void){
    int n, m;
    
    printf("Dame el primer numero a: ");
    scanf("%d", &n);
    printf("Dame el primer numero b: ");
    scanf("%d", &m);
    
    if (n*m == 100){
        printf("El producto %d * %d es igual a 100", n, m);
    } else {
        printf("El producto %d * %d es distinto de 100", n, m);
    }
    
    return 0;
}

File: traslate_py_C/example2.py
n = int(raw_input("Dame un numero: "))

m = int(raw_input("Dame otro numero: "))

if n * m == 100:
    print "El producto %d * %d es igual a 100" % (n, m)
else:
    print "El producto %d * %d es distinto de 100" % (n, m)
